pages/glossary.md

```diff
@@ -0,0 +1,157 @@
+- [SafeAuth demo app](https://github.com/safe-global/safe-core-sdk/tree/main/packages/auth-kit/example) on github.com
+- [Safe + Web3Auth = SafeAuth: Portable Safe Accounts with social login](https://safe.mirror.xyz/WKxK5FENvkT8BjpowJQAhokYzb22438zUCG3wUSWvjc) on safe.mirror.xyz
+
+## Safe Guard
+
+A Safe Guard is a smart contract that adds restrictions on top of the n-out-of-m scheme that Safe accounts offer, making checks before and after a Safe transaction is executed.
+
+See also:
+- [Safe Guards documentation](./smart-account-guards) on docs.safe.global
+- [Zodiac Guards](https://zodiac.wiki/index.php?title=Category:Guards) on zodiac.wiki
+- [Get the enabled Safe Guard](./sdk-protocol-kit/reference#getguard) and [enable a Safe Guard](./sdk-protocol-kit/reference#createenableguardtx) with the Safe{Core} SDK on docs.safe.global
+
+## Safe Module
+
+A Safe Module is a smart contract that adds functionality to Safe while separating module logic from Safe core contracts.
+
+See also:
+- [Safe Modules documentation](./smart-account-modules) on docs.safe.global
+- [Safe Modules repository](https://github.com/safe-global/safe-modules) on github.com
+- [Zodiac Modules](https://zodiac.wiki/index.php?title=Category:Module) on zodiac.wiki
+- [Get the enabled Safe Modules](./sdk-protocol-kit/reference#getmodules) and [enable a Safe Module](./sdk-protocol-kit/reference#createenablemoduletx) with the Safe{Core} SDK on docs.safe.global
+
+## Smart Account
+
+A smart account (also known as smart contract account) leverages the programmability of smart contracts to extend its functionality and improve its security in comparison with [externally-owned accounts](#externally-owned-account). Smart accounts are controlled by one or multiple externally-owned accounts or other smart accounts, and all its transactions have to be initiated by one of those.
+
+Some common features that smart accounts offer to their users are:
+- Multisignature scheme
+- Transaction batching
+- Account recovery
+- [Gasless transactions](#gasless-transaction)
+
+Safe is one of the most trusted implementations of a smart account.
+
+## Transaction
+
+A transaction is an action initiated by an [externally-owned account](#externally-owned-account) to update the state of the EVM network. Transaction objects need to be signed using the sender's private key, require a fee and must be included in a validated block.
+
+A Safe transaction is a transaction sent to a Safe Proxy contract calling the [execTransaction](https://github.com/safe-global/safe-smart-account/blob/main/contracts/Safe.sol#L104) method.
+
+See also:
+- [Transactions](https://ethereum.org/developers/docs/transactions) on ethereum.org
+
+## Threshold
+
+The threshold of a Safe account is a crucial configuration element that enables the use of Safe as a multisignature smart account. It defines the number of required confirmations from the Safe owners a Safe transaction must have to be executable.
```

Comment:
I think using `Safe transaction` helps understand the context better, highlighting it only for Safe transactions, and not regular ones. I know it might be redundant, but more clear in my opinion.

Reply:
Maybe we add it like '(Safe) transaction'?